test-harness/src/test/java/com/netflix/conductor/tests/utils/SqlServerTestModule.java

```diff
@@ -0,0 +1,98 @@
+/*
```

Comment:
This module doesn't appear to be used. In other modules it's used in integration tests (end to end).

Reply:
There are now integration tests for sqlserver
https://github.com/TwoUnderscorez/conductor/commit/44bcafaac894648992c2e16e754a252d35d8e79b